osf/migrations/0205_auto_20200402_1344.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.15 on 2020-04-02 13:44
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+import django_extensions.db.fields
+import osf.models.base
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('osf', '0204_ensure_schemas'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Brand',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('created', django_extensions.db.fields.CreationDateTimeField(auto_now_add=True, verbose_name='created')),
+                ('modified', django_extensions.db.fields.ModificationDateTimeField(auto_now=True, verbose_name='modified')),
+                ('name', models.CharField(blank=True, max_length=30, null=True)),
+                ('hero_logo_image', models.URLField()),
+                ('topnav_logo_image', models.URLField()),
+                ('hero_background_image', models.URLField()),
```

Comment:
so the idea is that someone uploads the assets on the provider, then copies the URLs for those assets into these text fields? it works, but is a clunky workflow...

wondering if it be feasible to do the asset upload on this page instead? or have a dropdown of existing assets to select from?

if that's a lot of work (i'm not too familiar with the admin, or with django forms...) it's fine as-is, probably

Reply:
This is something I discussed with @Johnetordoff. In order to have an asset upload on this page, we would probably need to change the brand model file fields from URLField to file fields, create a new storage type, etc. I wasn't sure if that was in the scope of this.